src/com/xilinx/rapidwright/edif/EDIFHierNet.java

```diff
@@ -141,11 +147,80 @@ public boolean equals(Object obj) {
 	 */
 	@Override
 	public String toString() {
-		return "EDIFHierNet [hierarchicalInstName=" + hierarchicalInst + ", net=" + net + "]";
+		return getHierarchicalNetName();
```

Comment:
@eddieh-xlnx - Just FYI - the previous toString() is not consistent with `EDIFHierCellInst` and `EDIFHierPortInst`.  Here, I just want the full hierarchical name.

Reply:
Good! I've been wanting to make this change too!